docs/src/main/sphinx/release/release-362.md

```diff
@@ -0,0 +1,83 @@
+# Release 362 (17 Sep 2021)
+
+## General
+
+* Add support for {func}`listagg`. ({issue}`4835`)
+* Improve join performance. ({issue}`8974`)
+* Improve performance of queries that contain `DISTINCT` clause. ({issue}`8967`, {issue}`9194`)
+* Improve query performance by reducing CPU overhead of repartitioning data across worker nodes. ({issue}`9102`)
+* Limit the number of backend SPI metadata calls when `java.sql.DatabaseMetaData` API is used to 
```

Comment:
This is talking about implementation details. What is the user impact?

Reply:
Let's make it:
```
Better performance of listing table columns via `java.sql.DatabaseMetaData` API when the filter is used on schema name.
```